packages/design-system/src/components/Combobox/Combobox.tsx

```diff
@@ -47,6 +47,7 @@ interface ComboboxProps
     Omit<ComboboxPrimitive.TextInputProps, 'required' | 'disabled' | 'value' | 'onChange' | 'size'> {
   clearLabel?: string;
   creatable?: boolean | 'visible';
+  creatableDisabled?: boolean;
```

Comment:
![screenshot 2025-05-05 at 11 11 25@2x](https://github.com/user-attachments/assets/dc8dfb0a-d429-44b7-8ad5-bd956ef94df7)

fyi the props table doesn't receive the type properly